### PR TITLE
Modified README.md about the shader requirements

### PR DESCRIPTION
**Pull request by ccpandhare**

### CODE REVIEW

**Comment by ccpandhare** [comment]
Hey, Does the README look good enough?

File: README.md [hunk 1]
```diff
@@ -51,8 +51,8 @@ var distorter = FisheyeGl({
     x: 1, // 0 to 2; default 1
     y: 1  // 0 to 2; default 1
   },
-  fragmentSrc: "../shaders/fragment.glfs", // these are provided in the /shaders/ directory
-  vertexSrc:   "../shaders/vertex.glvs"
+  fragmentSrc: "path/to/fragment.glfs", // defaults to an inbuilt fragment shader
```
Comment (jywarren) [inline README.md]: maybe say `// optional, defaults...`

Comment (ccpandhare) [inline README.md]: Shall do!

Comment (ccpandhare) [inline README.md]: Changed.

**Comment by jywarren** [comment]
Perfect! But boarding a plane now I wont be able to make the new npm
release for some hours.

**Pull request merged**